### PR TITLE
Notable changes for v2.0.5

### PR DESCRIPTION
**Pull request by Eirik0**

### CODE REVIEW

File: doc/release-notes.md [hunk 1]
```diff
@@ -4,3 +4,30 @@ release-notes at release time)
...
+
+New consensus rule: Reject blocks that violate turnstile
+--------------------------------------------------------
+In the 2.0.4 release the consensus rules were changed on testnet to enforce a
+consensus rule which marks blocks as invalid if they would lead to a turnstile
+violation in the Sprout or Shielded value pools.
+**This release enforces the consensus rule change on mainnet**
+
+The motivations and deployment details can be found in the accompanying
+[ZIP draft](https://github.com/zcash/zips/pull/210).
```
Comment (LarryRuane) [inline doc/release-notes.md]:
```suggestion
[ZIP draft](https://github.com/zcash/zips/pull/210) and
[PR 3968](https://github.com/zcash/zcash/pull/3968).
```

Comment (Eirik0) [reply doc/release-notes.md]: I made this change and removed the other reference to 3968.

**Comment by Eirik0** [comment]
I updated the ARM section with a suggestion from @daira. @mdr0id can you rereview when you get a chance?

File: doc/release-notes.md [hunk 2]
```diff
@@ -4,3 +4,47 @@ release-notes at release time)
...
+--------------------
+Added ARMv8 (AArch64) support. This enables users to build zcash on even more
+devices.
+
+For information on how to build see the [User Guide](https://zcash.readthedocs.io/en/latest/rtd_pages/user_guide.html#build)
+
+Users on the Zcash forum have reported successes with both the Pine64 Rock64Pro
+and Odroid C2 which contain 4GB and 2GB of RAM respectively.
+
+Just released, the Odroid N2 looks a great solution with 4GB of RAM. The newly
```
Comment (mdr0id) [inline doc/release-notes.md]: minor nit on grammer: `looks like a great`

**Comment by Eirik0** [comment]
@zkbot r+

**Pull request merged**